HealthcareSystem/Backend/Service/SendingMail/MailService.cs

```diff
@@ -34,6 +34,7 @@ public async Task SendWelcomeEmailAsync(WelcomeRequest request)
         private async Task SendEmail(MimeMessage email)
         {
             using var smtp = new SmtpClient();
+            smtp.CheckCertificateRevocation = false;
```

Comment:
What was the issue here? Maybe next time you have something like this that isn't really obvious / familiar, write a short explanation of what happened in the commit description or on the PR. It might be useful both to reviewers when reading this, and to you as future reference.

Reply:
[This ](https://stackoverflow.com/questions/59026301/sslhandshakeexception-an-error-occurred-while-attempting-to-establish-an-ssl-or) was a problem.